src/main/java/com/logentries/log4j2/LogentriesManager.java

```diff
@@ -22,15 +27,15 @@ static LogentriesManager getManager(String name, FactoryData data)
         @Override
         public LogentriesManager createManager(String name, FactoryData data)
         {
-            return new LogentriesManager(name,data);
+            return new LogentriesManager(new AsyncLoggerContext(name), name, data);
```

Comment:
I think we should not be forcing to use AsyncLogger - this can be done configuration/system setting wise too: https://logging.apache.org/log4j/2.x/manual/async.html